partials/attributes.adoc

```diff
@@ -7,6 +7,8 @@
 :PRODUCT_LONG: IBM Business Automation Manager Open Editions
 :OPENSHIFT: OpenShift
 :VERSION: 9.0.0
+:VERSION901: 9.0.1
+:VERSION90X : 9.0.x
```

Comment:
For consistency, please use VERSION_901 and VERSION_90X.

Reply:
Fixed